anet-dictionary.yml

```diff
@@ -1286,6 +1286,7 @@ fields:
     position:
       name: NATO Billet
       type: ANET User
+      positionRole: Position Role
```

Comment:
I would put this directly under `fields.position` since there's no need to have distinct labels for either advisor or principal.

Reply:
Applied